doc/source/ray-references/glossary.rst

```diff
@@ -70,6 +69,34 @@ Ray Data specifically, please see the :ref:`Ray Data Glossary<datasets_glossary>
         learning framework (eg. Torch, TensorFlow), used to set up distributed
         data-parallel training for :ref:`Ray Train’s built-in trainers<train-api>`.
 
+    Batch format
+        The way batches of data are represented.
```

Comment:
Now that this is in the global glossary can you clarify that this is refeerring to Ray Data in particular?

The way batches of data are represented in Ray datasets.